src/js/components/nft/nft-create-form.jsx

```diff
@@ -0,0 +1,382 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {withRouter} from 'react-router-dom';
+import {Button, Col, Form, Row} from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import * as format from '../../helper/format';
+import * as validate from '../../helper/validate';
+import ModalView from './../utils/modal-view';
+import * as text from '../../helper/text';
+import API from '../../api';
+import ErrorList from './../utils/error-list-view';
+import Transaction from '../../common/transaction';
+import HelpIconView from '../utils/help-icon-view';
+import {changeLoaderState} from '../loader';
+import ReactChipInput from 'react-chip-input';
+import ImageUploader from 'react-images-upload';
+
+
+class NftCreateForm extends Component {
+    constructor(props) {
+        super(props);
+        const propsState = props.location.state || {};
+
+        this.state = {
+            sending                 : false,
+            dns_valid               : false,
+            dns_validating          : false,
+            fee_input_locked        : true,
+            error_list              : [],
+            modal_show_confirmation : false,
+            modal_show_send_result  : false,
+            modal_body_send_result  : [],
+            address_base            : '',
+            address_version         : '',
+            address_key_identifier  : '',
+            amount                  : propsState.amount || '',
+            fee                     : '',
+            image                   : undefined,
+            destination_address_list: [],
+            txid                    : propsState.txid,
+            nft_src                 : propsState.src,
+            nft_hash                : propsState.hash
+        };
+
+        this.send = this.send.bind(this);
+    }
+
+    componentWillUnmount() {
+        if (this.state.sending) {
+            API.interruptTransaction().then(_ => _);
+        }
+    }
+
+    componentDidMount() {
+        let amount_default = 10000;
+        if (this.props.location?.state?.amount) {
+            amount_default = this.props.location.state.amount;
+        }
+        this.amount.value = format.millix(amount_default, false);
+    }
+
+    send() {
+        let error_list = [];
+        if (this.state.sending) {
+            API.interruptTransaction().then(_ => _);
+            this.setState({
+                canceling: true
+            });
+            return;
+        }
+        const transaction_param = {
+            addresses: validate.required('address', this.state.destination_address_list, error_list),
```

Comment:
addresses - don't use plural. use result_address or address_list. please fix other places where is is called too.
also in src/js/common/transaction.js:9 make sure variables are in lower case and you don't use plural either